etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/SendOrderUseCase.java

```diff
@@ -50,7 +50,8 @@ public void convertAndSend(final Order<?> order, String receivedSubmissionId)
                 EtorMetadataStep.ETOR_PROCESSING_TAG_ADDED_TO_MESSAGE_HEADER);
 
         String sentSubmissionId = sender.send(omlOrder).orElse(null);
-
         sendMessageHelper.saveSentMessageSubmissionId(receivedSubmissionId, sentSubmissionId);
+
+        sendMessageHelper.linkMessage(receivedSubmissionId);
```

Comment:
Do we want to wait until the very end to link?  Or do it right after saving the metadata given the received ID, which is towards the beginning?